spec/cljs/warm_places/geonames_api_spec.cljs

```diff
@@ -10,8 +9,8 @@
             [warm_places.geonames_api :refer [geonames-api-url
                                           call-geonames-api
                                           get-city-name
-                                          get-cities-with-temperatures
-                                          get-city-names
+                                          get-city-temperature-hashes
```

Comment:
no need to mention the 'hashes' part.  it's an implementation detail and subject to change if you decided to move the hashes into a module or something.